how-to/how-to-navigate-local-logs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How to Navigate to The Extension for Visual Studio and The Classic Windows Desktop Client Logs
```

Comment:
Again, I suggest using shorter names and different order
How to Navigate to the Logs for the AppBuilder VS Extension and Windows Client